Documentation/install.rst

```diff
@@ -1097,30 +1097,41 @@ If for some reason you do not want to run Cilium as a container image.
 Installing it from source is possible as well. It does come with additional
 dependencies described in :ref:`admin_system_reqs`.
 
+0. Download and install go-bindata
+
+    $ go get -u github.com/jteeuwen/go-bindata/...
+
 1. Download & extract the latest Cilium release from the ReleasesPage_
 
 .. _ReleasesPage: https://github.com/cilium/cilium/releases
 
 .. code:: bash
 
-    $ wget https://github.com/cilium/cilium/archive/v0.10.0.tar.gz
-    $ tar xzvf v0.10.0.tar.gz
-    $ cd cilium-0.10.0
+   $ go get github.com/cilium/cilium
+   $ cd $GOPATH/src/github.com/cilium/cilium
 
 2. Build & install the Cilium binaries to ``bindir``
 
 .. code:: bash
 
-   $ make
+   $ git checkout v0.11
+   $ # We are pointing to $GOPATH/bin as well since it's where go-bindata is
+   $ # installed
+   $ PATH=$GOPATH/bin:$PATH make
```

Comment:
I would move this to a requirements step in the beginning. `$GOPATH/bin` must be part of `$PATH`

Reply:
I've added it to the requirements, looks good?